src/python/python.cpp

```diff
@@ -432,6 +443,8 @@ PYBIND11_MODULE(onnxruntime_genai, m) {
       .def("is_done", &PyGenerator::IsDone)
       .def("compute_logits", &PyGenerator::ComputeLogits)
       .def("get_output", &PyGenerator::GetOutput)
+      .def("get_logits", &PyGenerator::GetLogits)
+      .def("set_logits", &PyGenerator::SetLogits)
```

Comment:
Do `get_logits()` and `set_logits()` work with float16 and float32 precision or only float32 precision?

Reply:
Always float32, as that's what our internal logits processing code uses. We convert whatever the model outputs to this to avoid needing multiple combinations of implementations for each type.